lib/companion_web/jobs/process_opt_out.ex

```diff
@@ -58,6 +59,8 @@ defmodule Companion.Jobs.ProcessOptOut do
       type: @type_nurse_optout,
       cmsisdn: extract_number_from_urn(nurse_urn),
       dmsisdn: device_msisdn,
+      eid: id,
```

Comment:
I think that `eid` is required to be a UUID, so we'll have to convert the database ID (which is an integer) into a UUID. Ecto has a built-in UUID tool which we can probably use for this https://hexdocs.pm/ecto/Ecto.UUID.html, something like:
```
Ecto.UUID.cast!(<<id::128>>)
```

Reply:
I'll try that thanks